packages/client/mutations/useGenericMutation.ts

```diff
@@ -0,0 +1,48 @@
+import {useMutation, UseMutationConfig} from 'react-relay'
+import useAtmosphere from '../hooks/useAtmosphere'
+//import SendClientSegmentEventMutation from './SendClientSegmentEventMutation'
+
+type Handlers = {
+  onSuccess?: (res: any, variables: any) => void
+  onError?: (error: any) => void
+}
+
+export const useGenericMutation = (
+  mutation: any,
+  dataField: string,
+  successMessage: string,
+  errorMessage: string
+  // TODO some segment config?
+) => {
+  const [commit, submitting] = useMutation(mutation)
+  const atmosphere = useAtmosphere()
+
+  const execute = (config: UseMutationConfig<any>, handlers?: Handlers) => {
```

Comment:
`config` already has `onError, onSuccess, variables`. Could we remove `handlers`?

Reply:
+1